pages/conduct.md

```diff
@@ -0,0 +1,40 @@
+<script>{
+	"title": "jQuery Foundation Code of Conduct",
+	"pageTemplate": "page-conduct.php"
+}</script>
+
+Like the technical community as a whole, the jQuery team and community is made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching, and connecting people.
+
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to founders, mentors and those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the jQuery Foundation and its projects. This includes IRC, the mailing lists, the issue tracker, jQuery events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it by emailing conduct@jquery.com. For more details please see our [Reporting Guidelines](https://jquery.org/conduct/reporting/)
+
+* **Be friendly and patient.**
```

Comment:
So here's a first pass, I'm open to suggestions:

"We understand that everyone does not have the same level of experience or knowledge, be it technical or non technical in nature.  Nobody knows everything, and we want to be a community where people can not only contribute, but feel comfortable to ask questions as well and perhaps learn along the way. This means that if someone is wrong or says something accidentally offensive, we don't mock or lash back at them.  Remember that we all were newbies at one point."